crates/biome_js_analyze/src/lint/correctness/no_unused_variables.rs

```diff
@@ -173,21 +151,7 @@ fn suggested_fix_if_unused(binding: &AnyJsIdentifierBinding) -> Option<Suggested
         AnyJsBindingDeclaration::JsArrowFunctionExpression(_) => {
             suggestion_for_binding(binding)
         }
-        AnyJsBindingDeclaration::TsPropertyParameter(_) => None,
-        AnyJsBindingDeclaration::JsFormalParameter(parameter) => {
-            if is_function_that_is_ok_parameter_not_be_used(&parameter.parent_function()) {
-                None
-            } else {
-                suggestion_for_binding(binding)
-            }
-        }
-        AnyJsBindingDeclaration::JsRestParameter(parameter) => {
-            if is_function_that_is_ok_parameter_not_be_used(&parameter.parent_function()) {
-                None
-            } else {
-                suggestion_for_binding(binding)
-            }
-        }
```

Comment:
This changes the behaviour of the rule, and it's a breaking change. We should change this only when the new rule goes out of nursery.

Reply:
I've reverted the changes to `noUnusedVariables` now